File: 0009-palindrome-number/0009-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        long long n = x,rev=0;
        while(n>0){
            long long digit = n%10;
            rev = rev*10+digit;
            n = n/10;
        }
        if(rev == x){
            return 1;
        }
        return 0;
    }
};

File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        int temp = x;
        int ld=0,rev=0;
        while(x>0){
            ld = x%10;
            rev = (rev*10)+ld;
            x/=10;
        }
        if(temp==rev){
            return true;
        } else{
            return false;
        }
    }
}